vernac/record.ml

```diff
@@ -621,6 +653,20 @@ let build_record_constant ~rdata ~ubind ~univs ~cumulative ~template fields para
   in
   List.map map inds
 
+(** [declare_class] will prepare a declare a [Class]. This is done in
+   2 steps:
+
+  - first, two markely different paths are followed depending on
+    whether the class declaration refers to a constant or to a record,
+    that is to say:
+
+      Class foo := bar.
```

Comment:
Definitional classes look more like `Class foo := bar : T.` which is equivalent to
~~~coq
Definition foo := T.
Definition bar (x:T) : foo := x.
Existing Class foo.
~~~
`Class foo := bar` is an error.

Reply:
Done thanks, sorry for the imprecision.

Reply:
Please re-check to see how it looks [don't hesitate to push a commit updating the comment]